packages/yoga/src/Stepper/native/Stepper.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { node } from 'prop-types';
+import styled from 'styled-components';
+
+import customPropType from '../customPropType';
+import Line from './Line';
+import Dots from './Dots';
+
+const Wrapper = styled.View`
+  width: 100%;
+`;
+
+const LineWrapper = styled.View(
+  ({ theme: { spacing } }) => `
+  width: 100%;
+  padding: 0 ${spacing.xxlarge}px;
+`,
+);
+
+/** Allows the user to navigate between steps relationed to a same context. */
+const Stepper = ({ children, activeStep, ...rest }) => (
+  <Wrapper {...rest}>
+    <LineWrapper>
+      <Line
+        activeStep={activeStep}
+        totalSteps={React.Children.count(children) - 1}
+      />
+      <Dots
+        activeStep={activeStep}
+        labels={React.Children.map(children, child => child.props.label)}
+      />
+    </LineWrapper>
+    {children[activeStep]}
+  </Wrapper>
+);
+
+Stepper.displayName = 'Stepper';
+
+Stepper.propTypes = {
+  /** Must be an Stepper.Step component. */
+  children: node,
```

Comment:
Why not [this](https://gist.github.com/gaurav5430/c491b7aba253a2e9c17a57c028ea92a7#file-custom-proptypes-js)?

Reply:
Done!